doc/03-Configuration.md

```diff
@@ -0,0 +1,63 @@
+# Configuration
+
+If Icinga Web has been installed but not yet set up, please visit Icinga Web and follow the web-based setup wizard.
+For Icinga Web setups already running, log in to Icinga Web with a privileged user and follow the steps below to
+configure Icinga Notifications Web:
+
+## Module Activation
+
+If you just installed the module, do not forget to activate it on your Icinga 2 Web instance(s) by using your
+preferred way:
+
+- Access the Icinga 2 command-line interface on your master(s) and execute `icingacli module enable notifications`.
+- Navigate to the Icinga 2 website(s), log in as a privileged user and activate the module under `Configuration →
+  Modules → Notifications` by switching the state from `disabled` to `enabled`.
+
+## Database Configuration
+
+Connection configuration for the database `notifications`, which both,
+the [Icinga Notifications Daemon](https://github.com/Icinga/icinga-notifications) and the [Icinga Notifications Web
+Module](https://github.com/Icinga/icinga-notifications-web), use.
+
+!!! tip
+If not already done, initialize your `notifications` database by following
+the [instructions](https://icinga.com/docs/icinga-notifications/latest/doc/02-Installation#setting-up-the-database).
+
+1. Create a new resource for the Icinga Notifications database via the `Configuration → Application → Resources` menu.
+2. Configure the resource you just created as the database connection for the Icinga Notifications Web module using the
+   `Configuration → Modules → notifications → Database` menu.
+
+## Channels Configuration
+
+As this module notifies contacts in case of events and incidents, you need to configure appropriate communication
+channels.
+Currently, the following technologies are supported:
+
+- E-Mail (SMTP)
+- Rocket.Chat
+- Webhook
+
+You need to configure at least one valid communication channel to be able to provide your contacts notifications.
+
+## Sources Configuration
+
+!!! question
+This paragraph is currently missing more specific information about adding external sources.
+
+The notifications module operates on data fed by miscellaneous sources and is therefore not restricted to Icinga 2 only.
+Though, any other external system needs to respect a specific data structure when providing its own data.
+
+You need to provide at least one valid source for this module to function properly.
+
+### Adding an Icinga 2 source
+
+If you want the notifications module to process Icinga 2 events, you will need to add it as a source:
+
+1. Navigate to `Configuration → Module → notifications → Sources` and add a new source.
+2. Choose type `Icinga` and provide Icinga 2 API credentials with the following
+   [permissions](https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/#overview):
+   - events/*
+   - objects/query/*
```

Comment:
@oxzi Are these correct?

Reply:
@nilmerg I double checked it on my installation and @yhabteab confirmed it to me as well.

Reply:
I have written the same over at my PR in #210, yes.

Please take a look at this (temporary) doc page: https://github.com/Icinga/icinga-notifications/blob/20e315af118ede51f0c37bf8540404307dfd0c82/doc/02-Installation.md#configuring-icinga-2

However, to avoid double-entry bookkeeping, I would leave it out of this documentation.